plugin.video.dailymotion_com/addon.xml

```diff
@@ -1,8 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="plugin.video.dailymotion_com" name="DailyMotion.com" version="2.3.5" provider-name="gujal">
+<addon id="plugin.video.dailymotion_com" name="DailyMotion.com" version="2.3.6" provider-name="gujal">
     <requires>
         <import addon="xbmc.python" version="2.1.0"/>
```

Comment:
Please fix `WARN: For jarvis it is advised to set xbmc.python version to 2.24.0` with the next update.

Reply:
Will do in the next update. Thanks for merging the PR